packages/storybook-integration/src/preset.ts

```diff
@@ -4,6 +4,7 @@ import {
   ImaCliArgs,
   compileLanguages,
 } from '@ima/cli';
+import { runImaPluginsHook } from '@ima/cli/dist/webpack/utils.js';
```

Comment:
As far as I know, it is prefered to import only directly from `@ima/cli`. If the key is not exported right now, it should be added to the `main.js` file in `@ima/cli`. Eventually, these kinds of imports might be even disabled.

Reply:
You are correct, I completely forgot about this one while debugging :D

Reply:
fixed